src/python/grpcio/grpc/_cython/_cygrpc/completion_queue.pyx

```diff
@@ -74,7 +68,41 @@ cdef class CompletionQueue:
     with self.poll_condition:
       self.is_polling = False
       self.poll_condition.notify()
+    return self.interpret_event(event)
+
+  def pluck(self, records.OperationTag tag, records.Timespec deadline=None):
+    # Plucking a 'None' tag is equivalent to passing control to GRPC core until
+    # the deadline.
```

Comment:
... and if the deadline is `None`, as is the case for the call site introduced in this function, what then?

Wait, it's not `None`, it's `records.Timespec(None)`. What's that?